### PR TITLE
Notice style gem update

### PR DESCRIPTION
**Pull request by sineade**
:art:

Added cart styling to the gem, because it is no longer only used in membership, it's also used in registries.

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

File: app/assets/stylesheets/ama_layout/layout_components/error-messages.scss [hunk 1]
```diff
@@ -1,10 +0,0 @@
-.notice_notification{
```
Comment (jbabe) [inline app/assets/stylesheets/ama_layout/layout_components/error-messages.scss]: Heads up to @sineade and @kathleenAMA - this file gets changed in the `registries` branch https://github.com/amaabca/ama_layout/compare/registries#diff-cb69a10c988591d1e7df02fb1414c86bR11

so we'll have a merge conflict when we try to merge registries in

Comment (jbabe) [inline app/assets/stylesheets/ama_layout/layout_components/error-messages.scss]: @kathleenAMA you'll have to re-add when you merge `master` into `registries`:

`git add app/assets/stylesheets/ama_layout/layout_components/error-messages.scss`

**Comment by kathleenAMA** [comment]
@jbabe Yeah once I merge master into registries I'll be moving the status classes into the new file that Sinead created.

**Pull request merged**